.github/workflows/run-e2e-tests.yml

```diff
@@ -28,3 +28,9 @@ jobs:
         aws-region: eu-west-1
     - name: Run integration tests
       run: npm run lerna-test:e2e
+    - name: Test packaging
+      run: | 
+        npm run lerna-package
+        cd examples/cdk
+        npm install ../../packages/**/dist/aws-lambda-powertools-*
```

Comment:
My understanding is that this install ALL packages from local and ALSO modify `package.json`? Why use the local version instead of the last version?

I tried running this and it only updates the `commons` package.  `logger` in `node_modules` is still 0.2.0. 

![image](https://user-images.githubusercontent.com/61999/149928001-65337ecf-3f3c-40b2-916e-d79e5021dae9.png)

![image](https://user-images.githubusercontent.com/61999/149928061-30672f7f-5696-4543-ace1-93ee1578dd2a.png)

If this isn't intentional, we can just install each module one by one.